modules/jupyter-widget/src/widget.js

```diff
@@ -87,6 +87,24 @@ export class DeckGLView extends DOMWidgetView {
     hideMapboxCSSWarning();
   }
 
+  onViewStateChange() {
+    // Saves viewport state out from the viewport set in the widget
+    // TODO need to determine when this is called -- autosave? Provide a button?
+    // Python function call?
+    const jsonConfig = JSON.parse(this.model.get('json_input'));
+    const viewport = this.jsonDeck.deckConfig.getViewports()[0];
```

Comment:
Note to self: See [here](https://github.com/uber/deck.gl/blob/master/docs/api-reference/deck.md)